### ISSUE TITLE
O4-06: specular gloss applied as specStrength *= glossSample.r rather than authored Phong-exponent multiplier

### ISSUE THREAD

**Issue by matiaszanolli** [matiaszanolli/ByroRedux#704]
**Severity**: LOW (defer behind PBR pipeline cleanup)
**File**: `crates/renderer/shaders/triangle.frag:728-734`
**Dimension**: Rendering Path

Gamebryo `NiTexturingProperty` slot 3 (gloss) is documented as a per-texel multiplier on `NiMaterialProperty.shininess` (i.e. the Phong exponent), NOT on the specular strength.

The current shader reads `glossSample.r` and multiplies `specStrength`. Effect: gloss-masked surfaces (polished armor inserts on dull leather backings) come out with the right ON/OFF mask but slightly wrong roughness profile.

Not visible on most Oblivion content because `glossiness` typically defaults to 80 and the mask binarises to 0/1 anyway.

**Fix**: Route `glossSample.r` to a per-pixel `roughness` modulation. Defer behind PBR roughness-pipeline cleanup.

## Completeness Checks
- [ ] **SIBLING**: same pattern checked in related files (other shader types, other block parsers, other game variants)
- [ ] **TESTS**: regression test added for this specific fix
- [ ] **CROSS-GAME**: if Oblivion-only fix, verify FO3/FNV/Skyrim variants are unaffected
- [ ] **DOC**: ROADMAP.md / CLAUDE.md / audit-oblivion.md updated if they cite the affected behaviour

---
*From [AUDIT_OBLIVION_2026-04-25.md](docs/audits/AUDIT_OBLIVION_2026-04-25.md) (commit 1ebdd0d)*

**Comment by matiaszanolli** [comment]
Fixed in 1129287.

Per Gamebryo 2.3 `Include/NiStandardMaterial.h`:
```
virtual bool HandleGlossMap(Context&, NiMaterialResource* pkUVSet,
                            NiMaterialResource*& pkGlossiness);
```
the slot-3 sample feeds the **glossiness / shininess** (Phong exponent) channel — confirming the issue's claim against the legacy source. The fragment shader at `triangle.frag:780-789` now routes `glossSample.r` through `roughness = mix(1.0, roughness, glossSample)` — gloss = 1 keeps the authored roughness, gloss = 0 lerps to fully rough — and `specStrength` stays at the material's authored constant.

Practical effect: polished metal trim now produces a sharp specular highlight while the surrounding dull leather strap keeps a broad/diffuse one (the lobe SHAPE varies across the mesh, not just the intensity). Most vanilla Oblivion content won't visibly change because `glossiness` defaults to 80 and gloss masks tend to binarise to 0/1, but the shading is now faithful to the legacy semantic and any content that authors a meaningful gradient in the slot-3 texture will benefit.

Doc-comments updated in lockstep on `gloss_map` (`core/ecs/components/material.rs`), `gloss_map_index` (`renderer/vulkan/context/mod.rs` + `renderer/vulkan/scene_buffer.rs`).

Recompiled `triangle.frag.spv`; 104 renderer lib tests pass; full workspace `cargo test` green; warnings unchanged.
